easybuild/easyblocks/generic/configuremake.py

```diff
@@ -350,13 +351,13 @@ def test_step(self):
         """
 
         test_cmd = self.cfg.get('test_cmd') or DEFAULT_TEST_CMD
-        if self.cfg['runtest'] or test_cmd != DEFAULT_TEST_CMD:
-            cmd = ' '.join([
-                self.cfg['pretestopts'],
-                test_cmd,
-                self.cfg['runtest'],
-                self.cfg['testopts'],
-            ])
+        run_test = self.cfg['runtest']
+        if run_test or test_cmd != DEFAULT_TEST_CMD:
+            # Make run_test a string (empty if it is e.g. a boolean)
+            if not isinstance(run_test, str):
```

Comment:
Don't use `str` directly, use `string_type` from `py2vs3`:

```python
from easybuild.tools.py2vs3 import string_type
...
             if not isinstance(runtest, string_type):

Reply:
I wanted to stick to what is used at other places in the same file, e.g. line [331](https://github.com/easybuilders/easybuild-easyblocks/blob/077a9036be1b5666a50bef1b746ec7834f256900/easybuild/easyblocks/generic/configuremake.py#L331)

Shall I still replace this and maybe all others in this file too?

Reply:
It's better to use `string_type` consistently, there could be surprises when using `str` w.r.t. behavior in Python 2 vs 3...

Reply:
Ok, done